packs/data/iconics.db/fumbus-level-5.json

```diff
@@ -2772,137 +2772,6 @@
             },
             "type": "feat"
         },
-        {
```

Comment:
How did this get deleted?

Reply:
I totally forgot to include this change in the commit write up. Given this was the one instance I knew in the compendium of an outdated version of the Bomber field, I manually updated Fumbus to use the updated version of the bomber field, by dropping Fumbus to 4th level and then back to 5th.

Reply:
Ah, gotcha. Sounds good, then!